src/qml/androidcustomdatadir.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2023 The Bitcoin Core developers
```

Comment:
nit 
```suggestion
// Copyright (c) 2024 The Bitcoin Core developers
```
or maybe even `2023-present`, which seems to be used in the main repo now.

Reply:
good catch...